moto/swf/exceptions.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import unicode_literals
+
+from boto.exception import JSONResponseError
+
+
+class SWFClientError(JSONResponseError):
```

Comment:
Yup, that is the goal. Raising a `werkzeug.exceptions.HTTPException` will have the response return a `4xx` status code back to the boto client which will then raise the `SWFResponseError`.